optimum/exporters/tflite/model_configs.py

```diff
@@ -27,3 +27,73 @@ class BertTFLiteConfig(TextEncoderTFliteConfig):
     @property
     def inputs(self) -> List[str]:
         return ["input_ids", "attention_mask", "token_type_ids"]
+
+
+class AlbertTFLiteConfig(BertTFLiteConfig):
+    pass
+
+
+class ConvBertTFLiteConfig(BertTFLiteConfig):
+    pass
+
+
+class ElectraTFLiteConfig(BertTFLiteConfig):
+    pass
+
+
+class RoFormerTFLiteConfig(BertTFLiteConfig):
+    pass
+
+
+class MobileBertTFLiteConfig(BertTFLiteConfig):
+    pass
+
+
+class XLMTFLiteConfig(BertTFLiteConfig):
+    pass
+
+
+class DistilBertTFLiteConfig(BertTFLiteConfig):
+    @property
+    def inputs(self) -> List[str]:
+        return ["input_ids", "attention_mask"]
+
+
+class MPNetTFLiteConfig(DistilBertTFLiteConfig):
+    pass
+
+
+class RobertaTFLiteConfig(DistilBertTFLiteConfig):
+    pass
+
+
+class CamembertTFLiteConfig(DistilBertTFLiteConfig):
+    pass
+
+
+class FlaubertTFLiteConfig(BertTFLiteConfig):
+    pass
+
+
+class XLMRobertaTFLiteConfig(DistilBertTFLiteConfig):
+    pass
+
+
+# TODO: no TensorFlow implementation, but a Jax implementation is available.
+# Support the export once the Jax export to TFLite is more mature.
```

Comment:
Why is it more mature? @michaelbenayoun

Reply:
Because it is an experimental feature: https://www.tensorflow.org/lite/api_docs/python/tf/lite/TFLiteConverter#experimental_from_jax

Also we need to add support for Jax in `optimum` for that.